website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -44,57 +89,235 @@ The [Whole Genome Germline Single Sample workflow](https://github.com/broadinsti
+| [Calling.HaplotypeCaller_GATK35_GVCF (HaplotypeCallerGATK3)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/GermlineVariantDiscovery.wdl) | PrintReads, HaplotypeCaller | GATK4, GATK3.5 | If `use_gatk3_haplotype_caller` is true, will call GATK3 Haplotypecaller to call variants in GVCF mode, otherwise will use the HaplotypeCaller_GATK4_VCF task below. |
+| [Calling.HaplotypeCaller_GATK4_VCF (HaplotypeCallerGATK4)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/GermlineVariantDiscovery.wdl) | HaplotypeCaller | GATK4 | If `use_gatk3_haplotype_caller` is false, will call GATK4 Haplotypecaller to call variants in GVCF mode. If `run_dragen_mode` is true, uses the --dragstr-params-path containing the DragSTR model and runs it with HaplotypeCaller in --dragen-mode. |
+| [Calling.DragenHardFilterVcf (DragenHardFilterVcf)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/GermlineVariantDiscovery.wdl) | VariantFiltration | GATK | If `dragen_mode_hard_filter` is true, performs hard filtering that matches the filtering performed by the DRAGEN 3.4.12 pipeline. |
+| [BamProcessing.SortSam (SortBamout)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/BamProcessing.wdl) | SortSam | Picard |If the option to make a BAM out file is selected ( `make_bamout` is true), sorts and gathers the BAM files into one file. |
+| [MergeBamouts](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/variant_calling/VariantCalling.wdl) | merge, index | samtools | If `make_bamout` is true, makes corrections to the merged BAM out file from Picard. | 
+| [Calling.MergeVCFs (MergeVCFs)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/GermlineVariantDiscovery.wdl) | MergeVcfs | Picard | Combines by-interval (g)VCFs into a single sample (g)VCF file. |
+| [QC.ValidateVCF (ValidateVCF)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/Qc.wdl) | ValidateVariants | GATK | Validates the (g)VCF from HaplotypeCaller with the -gvcf parameter. |
+| [QC.CollectVariantCallingMetrics (CollectVariantCallingMetrics)](https://github.com/broadinstitute/warp/blob/master/tasks/broad/Qc.wdl) | CollectVariantCallingMetrics | Picard | Performs quality control on the (g)VCF. |
+ 
+## Workflow outputs
+
+The table below describes the final workflow outputs. If running the workflow on Cromwell, these outputs are found in the respective task's execution directory.
+
+
+| Output variable name | Description | Type |
+| --- | --- | --- |
+| quality_yield_metrics | The quality metrics calculated for the unmapped BAM files. | Array of files |
+| unsorted_read_group_base_distribution_by_cycle_pdf | PDF of the base distribution for each unsorted, readgroup-specific BAM. | Array of PDFs |
```

Comment:
This is probably a dumb question, but is an array of pdfs different than an array of files? Did we just want to note the specific file type here? (not suggesting this needs to change, just curious)